graphql/.eslintignore

```diff
@@ -0,0 +1,9 @@
+config
+`**/node_modules`
+'.cache'
+'**/.cache'
+'**/build'
+'**/dist'
+'**/.storybook'
+'**/storybook-static'
+.eslintrc.cjs
```

Comment:
Three different conventions here, one without quotes, one with `'` quotes and one with ` quotes.
Probably all the quotes can be removed according to documentation [#1](https://eslint.org/docs/latest/use/configure/ignore) and [#2](https://git-scm.com/docs/gitignore):
> Globs are matched using [node-ignore](https://github.com/kaelzhang/node-ignore), so a number of features are available:
> - Lines beginning with # are treated as comments and do not affect the ignore patterns.
> - Paths are relative to the current working directory. This is also true of paths passed in via the --ignore-pattern [command](https://eslint.org/docs/latest/use/command-line-interface#--ignore-pattern).
> - Lines preceded by ! are negated patterns that re-include a pattern that was ignored by an earlier pattern.
> - Ignore patterns behave according to the .gitignore [specification](https://git-scm.com/docs/gitignore).

Reply:
Is this the only thing blocks the merging of this PR. This PR is base for 4 others, so I don't want to have this as a blocker, since this has been here for ages. I can have a new trivial PR for this when the rest are merged